tools/packages.yaml

```diff
@@ -6361,6 +6366,11 @@ url: https://github.com/periscop/openscop/releases/
 activity: low
 ---
 kind: url
+name: ots
+url: https://github.com/khaledhosny/ots
```

Comment:
This url should point to the releases or tags.  It indicates where updates can be downloaded.